docs/README.md

```diff
@@ -58,12 +58,12 @@ First, declare the `autocomplete-result-list.v2` block as a child block of the [
 | `maxHistory` | `number`   | Maximum number of items in the search history list.         | `5`   |
 | `maxSuggestedProducts` | `number` | Maximum number of items in the suggested products list.    | `3`  |
 | `maxSuggestedTerms`   | `number`  | Maximum number of items in the suggested terms list.  | `3` |
-| `autocompleteWidth` | `number`    | Autocomplete list width. The value must be between `0` and `100`   | `undefined`  |
+| `autocompleteWidth` | `number`    | Autocomplete list width (in percent). The value must be between `0` and `100`   | `undefined`  |
 | `productLayout` | `enum` | Defines the suggested products list layout when rendered. Possible values are `HORIZONTAL` and `VERTICAL`.  | `undefined` |
-| `hideTitles` | `boolean` | Defines whether all component titles are hidden when redered (`true`) or not (`false`).   | `false`  |
+| `hideTitles` | `boolean` | Defines whether all component titles are hidden when rendered (`true`) or not (`false`).   | `false`  |
 | `historyFirst` | `boolean` | Defines whether the search history list should be prioritized over the other lists (`true`) or not (`false`).  | `false`       |
 | `customBreakpoints` | `object` | Defines a maximum number of suggested products by breakpoints. Possible values are `md`, `lg` or `xlg`. | -             |
-| `__unstableProductOriginVtex` | `boolean` | Defines whether any Product Summary blocks' props should come with `null` value (`true`) or not (`false`). This prop because some product information does not come by default in the Search. | `false` |
+| `__unstableProductOriginVtex` | `boolean` | Some product information is not sent by default by the Intelligent Search to improve performance, in some cases this can generate an error. In case there is an error for not having information about the product or if you have sellers whitelabels, you can use this prop as `true` to retrieve the catalog data. | `false` |
```

Comment:
```suggestion
| `__unstableProductOriginVtex` | `boolean` |  When set as `true`, the prop enables the Intelligent Search to fetch catalog data in scenarios where product information is missing. This prop is extremely useful if you have [sellers White Labels](https://help.vtex.com/tutorial/what-are-franchise-account-and-seller-white-label--5orlGHyDHGAYciQ64oEgKa?locale=en#what-is-a-seller-white-label) or if you just want to avoid rendering errors related to product data. | `false` |
```

Reply:
Nice, but I think it is important to make it clear to the reader also that using this prop as `true` can decrease search performance.